llama.cpp

```diff
@@ -936,6 +936,12 @@ static void llama_model_load_internal(
             case 40: model.type = e_model::MODEL_13B; break;
             case 60: model.type = e_model::MODEL_30B; break;
             case 80: model.type = e_model::MODEL_65B; break;
+            default:
+                {
+                    if (hparams.n_layer < 32) {
+                        model.type = e_model::MODEL_7B;
```

Comment:
we are adding the even smaller 3B (OpenLLaMa) here https://github.com/ggerganov/llama.cpp/pull/1588